lib/core/src/Cardano/Wallet.hs

```diff
@@ -1875,11 +1881,13 @@ mkTxMeta interpretTime blockHeader wState tx cs expiry =
     let
         amtOuts =
             sum (mapMaybe ourCoins (outputs cs))
+            + fromIntegral (CoinSelection.deposit cs)
```

Comment:
@KtorZ

Reply:
Yes? What/why is this change?

Reply:
I wanted to know if this is correct, because of your comment https://github.com/input-output-hk/cardano-wallet/pull/2327#discussion_r532431872

Reply:
It is not. The current code is doing just fine :+1:

Reply:
(This calculation happens on the transaction outputs directly, so the deposit is already _implicitly_  included in the difference between inputs and outputs).

Reply:
Although, the point of this PR / Story is now to make it possible to identify whether the delta (in - out) is purely fee, or fee and deposits.